spark_ec2.py

```diff
@@ -327,13 +411,19 @@ def parse_args():
     parser.add_option(
         "--instance-profile-name", default=None,
         help="IAM profile name to launch instances under")
+    parser.add_option("-c", "--conf-file", default=None,
+                        help="Specify config file", metavar="FILE")
```

Comment:
I just realized we haven't merged #34 yet - Will it be fine if I merge that first ?